### PR TITLE
Set up spring boot application

### PR DESCRIPTION
**Pull request by Leks707**

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.4.1-bin.zip
```
Comment (VitaliiSerdiuk) [inline gradle/wrapper/gradle-wrapper.properties:3]: It may be 6.5

File: src/main/java/org/vov/farmer/controllers/HealthCheck.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/")
+public class HealthCheck {
+
+    @GetMapping("health")
+    public String healthCheck() {
+        return "Hello world";
```
Comment (VitaliiSerdiuk) [inline src/main/java/org/vov/farmer/controllers/HealthCheck.java:13]: Some test needed

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (Leks707) [reply gradle/wrapper/gradle-wrapper.properties:3]: updated

File: src/main/java/org/vov/farmer/controllers/HealthCheck.java (hunk 2 above)
Comment (Leks707) [reply src/main/java/org/vov/farmer/controllers/HealthCheck.java:13]: added

**Pull request merged**